src/cljam/io/fasta_index/core.clj

```diff
@@ -54,6 +61,7 @@
          (.indices fai)))))
 
 (defn get-indices
+  "Returns all indexes with :name val"
```

Comment:
Good documentation, but it could be even better if rewritten with a focus on explaining the differences from `get-headers`.